### ISSUE TITLE
[Feature] [Accessibility] - Expandable table to update

### ISSUE THREAD

**Issue by sLorenzini** [PrestaShopCorp/puik#363]
# What problem does this feature solve?

To improve accessibility on components, some tests are made on several components.
Expandable table component has got some changes into code tu update accessibility. There is a critical and incomplete accessibility tests.

![Capture d’écran 2024-07-18 à 16 07 53](https://github.com/user-attachments/assets/9a16ac46-d5e3-487d-88fd-5483d2deed72)

- Critical: header is empty

![Capture d’écran 2024-07-18 à 16 10 46](https://github.com/user-attachments/assets/72ba53a1-6182-495f-b6db-c20cb61d9494)

- Incomplete: <th> should have the same content as its data

![Capture d’écran 2024-07-18 à 16 10 53](https://github.com/user-attachments/assets/bb4e15ff-8856-414d-be64-a38c6f268052)

# What is your idea for solve this issue

Find a way to improve code with good values.
